ESMX_SingleModelInJuliaProto/SiMoCoJulia/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+cmake_minimum_required(VERSION 3.19)
+
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
+find_package(ESMF 8.6.0 MODULE REQUIRED)
```

Comment:
The NUOPC C interface wasn't added until after 8.6.0 (and also isn't in the patch 8.6.1)

Reply:
Very good point. I just pushed a fix.